docker-compose.override.yml

```diff
@@ -12,3 +12,18 @@ services:
       - app
     volumes:
       - .:/app
+
+  drush:
+    image: itkdev/drush8:latest
+    networks:
+      - app
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    depends_on:
+      - mariadb
+    entrypoint:
+      - drush
+    environment:
+      - PHP_MEMORY_LIMIT=512M
+    volumes:
+      - .:/app
```

Comment:
Why do you want this? `drush` is installed with composer (https://github.com/eReolen/base/blob/develop/composer.json#L11).

Reply:
When I run itkdev-docker-compose it could not find drush